py-polars/polars/expr/binary.py

```diff
@@ -206,17 +206,17 @@ def encode(self, encoding: TransferEncoding) -> Expr:
         ... )
         >>> colors.with_columns(
         ...     pl.col("code").bin.encode("hex").alias("code_encoded_hex"),
-        ... )
+        ... )  # doctest: +IGNORE_RESULT
```

Comment:
I couldn't get these special characters right. Will check later - either improve the display of these binary types or we can change the example.

Reply:
In my terminal it looks like below, but when I copy paste it into the example the spacing is off. Not sure what is going on. Have to look a bit more closely.

Indeed this is a nice improvement over just displaying `[binary data]`!

![image](https://github.com/pola-rs/polars/assets/3502351/e604d3da-e283-45e0-90a0-991bab641af8)